apps/sertra/components/HeadWrapper.tsx

```diff
@@ -0,0 +1,18 @@
+import Head from "next/head";
+
+interface HeadProps {
+  title: string;
```

Comment:
This is misleading and could easily be assumed to be user-visible title. `metaTitle` would be more fitting. Same for `description`, `metaDescription` would be more clear if you just see the call site.